app/code/Magento/InventoryBundle/Test/Api/BundleProductShouldBeInStockOnCreateTest.php

```diff
@@ -81,9 +82,7 @@ public function testIsBundleProductWithSimpleProductInStockAfterCreate()
             ],
             "extension_attributes" => [
                 "bundle_product_options" => $bundleProductOptions,
-                'stock_item' => [
-                    'is_in_stock' => 1,
-                ],
+                'stock_item' => $this->getStockItemData()
```

Comment:
why do we need to specify all the StockItem data? why does fallback not work?

Reply:
@maghamed I order to pass SOAP validation, we have to specify all these fields  as we create bundle product with stock item on the fly. Looks like it's been always that way:
https://github.com/magento-engcom/msi/blob/f2fdae5246c1b859e4ce7532458dd1243eff0991/dev/tests/api-functional/testsuite/Magento/Catalog/Api/ProductRepositoryInterfaceTest.php#L340